noir-projects/aztec-nr/aztec/src/encrypted_logs/payload.nr

```diff
@@ -13,13 +13,52 @@ use crate::{
     keys::point_to_symmetric_key::point_to_symmetric_key,
 };
 
-pub fn compute_encrypted_log<let P: u32, let M: u32>(
+pub fn compute_encrypted_note_log<let P: u32, let M: u32>(
```

Comment:
Added `compute_encrypted_note_log` and `compute_encrypted_event_log` wrapper functions which just call `compute_encrypted_log` with different offset.